res/css/_common.pcss

```diff
@@ -65,8 +68,7 @@ body {
     font-size: $font-15px;
     background-color: $background;
     color: $primary-content;
-    border: 0px;
```

Comment:
Having looked at the index.html file, I don't see this being declared inline there. I think that we would need to keep this line (unless it's applied somewhere else that I'm missing)

Reply:
It's useless string.